src/ansys_sphinx_theme/announcement_layout.html

```diff
@@ -0,0 +1,16 @@
+{% if theme_switcher %} 
+<div class="container-md">
+    <div class="row"></div>
+        <div id="header-announcement-spacer"></div>
+        <div class="col-lg-12 col-sm-6" id="header-announcement">
+            <p id="announce"></p>
+            <script>
+            let file = "announcement.html"
+            fetch (file)
+            .then(res => res.text())
+            .then(y => document.getElementById("announce").innerHTML = y);
+            </script>
```

Comment:
This script should:

* Check if the file exists.
* If not, do not add any content.
* Ideally, use relative path for the file.